lib/nero/wildlife_watch/wildlife_watch_estimate.rb

```diff
@@ -0,0 +1,29 @@
+require 'date'
+require 'set'
+
+module Nero
+  module WildlifeWatch
+    class WildlifeWatchEstimate < SimpleDelegator
+      def votes
+        data["votes"] ||= []
+        data["votes"]
+      end
+
+      def add_vote(vote)
+        self.votes << vote
+      end
+
+      def retired?
+        grouped_votes = votes.group_by {|i| i }
+        grouped_votes.default = []
+
+        return :human       if grouped_votes["human"].size >= 1                    # if one classification says it's a human
+        return :five_blanks if grouped_votes["blank"].size >= 5                    # if 5 classifications say the image is blank
+        return :consensus   if grouped_votes.any? { |key, group| group.size >= 7 } # if 7 classifications agree
```

Comment:
Unused block argument - `key`. If it's necessary, use `_` or `_key` as an argument name to indicate that it won't be used.